index.html

```diff
@@ -17,27 +17,36 @@
 </head>
 
 <body>
-  <div class="container-fluid">
-    <div class="container">
+  <div class="container-fluid sticky-top">
+    <div class="container top">
       <div class="row">
         <div class="col-3">
             <h1 class="logo-heading">Fun Bus</h1>
         </div>
-        <div class="col-9">
-          <nav class="nav">
-            <a class="nav-link active" href="#">Home</a>
-            <a class="nav-link" href="#">About Us</a>
-            <a class="nav-link" href="#">Blog</a>
-            <a class="nav-link" href="#">Contact</a>
-          </nav>
+        <div class="col-9 d-flex align-items-center justify-content-end navigation">
+          <!-- JavaScript Component: NavBar -->
+          <nav class="navbar navbar-expand-lg navbar-light bg-light">
+             <a class="navbar-brand" href="#">Site Navigation</a>
```

Comment:
With most navigation, it is not necessary to label it as such. Makes for a cleaner, more professional design. Not a big deal, just fyi.